docs/getting-started/managing-octopus-subscriptions.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Managing Octopus subscriptions
+description: Control Center is where you manage your Octopus subscriptions and their associated user access. 
+position: 40
+---
+
+Control Center is where you manage your Octopus subscriptions and their associated user access. 
+
+There are two types of Octopus subscriptions:
+
+1. **Cloud Instances**: Deployments as-a-service
+2. **Server Licenses**: Octopus on your infrastructure
+
+This page focuses on Control Center V2, our new system for managing Octopus subscriptions.
+
+## Control Center V1 and V2 
+
+### Explaining the two versions of Control Center
+
+In October 2022, we made the transition to use Stripe as our payment gateway. This decision required a new system for integration with Stripe. This new system is referred to as Control Center V2, and will be the future of Control Center. During 2023, subscriptions from V1 will be migrated to V2 as they are paid or renewed. Once that migration is complete, V1 will be deprecated.
```

Comment:
```suggestion
In October 2022, we moved to Stripe as our payment gateway, so we needed a new system for integration with Stripe. We refer to the new system as Control Center V2, and it will replace Control Center later this year. During 2023, we'll migrate your subscriptions from V1 to V2 as you pay or renew. After the migration is complete, we'll deprecate V1.
```